src/utils/test-runner.js

```diff
@@ -47,33 +28,61 @@ function validSourceApiVersion() {
     }
 }
 
+function getJestPath() {
+    const packageJsonPath = require.resolve('jest/package.json');
+
+    const { bin } = require(packageJsonPath);
+    return path.resolve(path.dirname(packageJsonPath), bin);
+}
+
+function getJestArgs(argv) {
+    // Extract known options from parsed arguments
+    const knownOptions = Object.keys(argv)
+        .filter((key) => argv[key] && JEST_PASSTHROUGH_OPTIONS.has(key))
```

Comment:
Looking through this repository, it doesn't look like we have any logic that requires explicit default values of `false` for boolean arguments so I think we can remove the defaults from the configuration in [this file](https://github.com/salesforce/sfdx-lwc-jest/blob/2c906c7f77342e74fbe93fbad1983bcf8a6728d9/src/options/options.js). That would make this code more robust since it wouldn't need to change if we ever introduce new non-boolean arguments with truthy values.

Otherwise, my second preference would be to change this to:
```suggestion
        .filter((key) => argv[key] !== false && JEST_PASSTHROUGH_OPTIONS.has(key))
```